### DISCUSSION TITLE
Hi, The new pipe Latest apk 0.27.4 Not working. Please fix the bug asap.

### DISCUSSION THREAD

**Discussion by vphanibhargav** [TeamNewPipe/NewPipe#11804] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Hi, The new pipe Latest apk 0.27.4 Not working. Please fix the bug asap.

### Additional information

Hi, The new pipe Latest apk 0.27.4 Not working. Please fix the bug asap.

**Comment by ShareASmile** [comment]
This is not the place to report a bug. Discussions are for some mild questions related to the app.

As for as your issue is concerned, See Pinned Issue at the top of Issue tracker.

Please follow the template & always check for duplicates first when opening an issue or a discussion next time.
